### ISSUE TITLE
Error: An argument named "response_headers_policy_id" is not expected here.

### ISSUE THREAD

**Issue by mattdodge** [cloudposse/terraform-aws-cloudfront-s3-cdn#214]
## Describe the Bug
Applying a rather simple configuration yields the following error
```
An argument named "response_headers_policy_id" is not expected here.
```
```hcl
module "files_cdn" {
  source  = "cloudposse/cloudfront-s3-cdn/aws"
  version = "0.81.0"

  environment = var.env
  namespace   = "myapp"
  name        = "assets"
}
```

It seems that, based on the [release notes](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v3.64.0), the `response_headers_policy_id` argument was added to the AWS provider in v3.64.0. The current requirements list the AWS provider requirement as `>= 3.41.0`. 

I'm not sure if this can be an optional argument or if you just want to bump the required AWS provider version up. But this threw me off when trying to use this module.

## Environment (please complete the following information):
```
Terraform v1.0.8
on darwin_arm64
+ provider registry.terraform.io/hashicorp/aws v3.60.0
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.5.0
+ provider registry.terraform.io/hashicorp/local v2.1.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/time v0.7.2
+ provider registry.terraform.io/terraform-aws-modules/http v2.4.1
```

**Comment by rvmey** [comment]
I'm having the same problem.

**Comment by mattdodge** [comment]
The workarounds I've found are:
* Upgrade the aws provider to 3.64.0
* Downgrade the cloudposse/cloudfront-s3-cdn/aws module to 0.75.0

You should only need to do one of these I think

**Comment by rvmey** [comment]
Thanks.  I upgraded the aws provider and it's working.  
```
version = "~> 3.64"
```

**Comment by rusowyler** [comment]
Versión 0.82.3 solves this problem (https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/releases/tag/).

**Comment by Nuru** [comment]
Closed by #218
